moped-editor/src/utils/taskOrderComponentHelpers.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { TextField } from "@material-ui/core";
+
+/*
+ * Socrata Endpoint
+ */
+export const SOCRATA_ENDPOINT =
+  "https://data.austintexas.gov/resource/ggav-ufvc.json?$limit=9999";
```

Comment:
Will this dataset graduate to become the production dataset? If so, 👍, and if not, I'm curious to learn what the mechanism will be to configure this URL based on environment -- I'm not sure what convention is used for that in the application currently.

Reply:
Yes, I anticipate that once we are automatically updating this dataset with an ETL process, we'll go ahead and drop the `[TEST] ` prefix in the title. I don't think we'll need to worry about maintaining multiple version of the Socrata dataset depending on environment.